src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
```

Comment:
@amb26 thoughts on appropriate way to manage something like a global toggle ("use native widgets" vs. "use jQuery widgets")? As I work my way through UIO and the Preferences Framework I'm finding several components that need to have a sense of the preference for the native vs. jQuery slider (primarily in terms of what HTML template they use). It doesn't seem appropriate to expect a user to override each one by one.

It seems at first glance reading the documentation on context awareness that it might be appropriate to make use of it - there are numerous components that need to respond to a context of "use native widgets" vs. "use jQuery widgets". Or is that more complexity than is necessary in this scenario? I'm conscious that whatever the approach taken is, it needs to avoid anything that seems like magic or central control fragility (hence my hesitation around the very idea of a "global toggle").